web/src/app/services/navigation/navigation.service.ts

```diff
@@ -186,7 +180,7 @@ export class NavigationService {
     taskId: string
   ) {
     this.router.navigateByUrl(
-      `${NavigationService.SURVEY_SEGMENT}/${surveyId}/${NavigationService.LOI_SEGMENT}/${loiId}/${NavigationService.SUBMISSION_SEGMENT}/${submissionId}/${NavigationService.TASK_SEGMENT}/${taskId}`
+      `${SURVEY_SEGMENT}/${surveyId}/${LOI_SEGMENT}/${loiId}/${SUBMISSION_SEGMENT}/${submissionId}/${TASK_SEGMENT}/${taskId}`
```

Comment:
FYI: Using taskId to identify the result will become problematic once we allow repeated tasks. That's out of scope for this PR, just wanted to call it out for future changes.

Reply:
That's good to know! Currently the taskId is used just to highlight the task